GeneratedRegexPatterns/MediumConfidenceSecurityModels.json

```diff
@@ -10,6 +17,16 @@
     ],
     "DetectionMetadata": "HighEntropy, MediumConfidence"
   },
+  {
+    "Pattern": "https?:\\/\\/(?:[^:@]+):(?<refine>[^:@?]+)@",
+    "Id": "SEC101/127",
+    "Name": "UrlCredentials",
+    "Signatures": [
+      "http",
+      "https"
+    ],
+    "DetectionMetadata": "MediumConfidence"
```

Comment:
>MediumConfidence [](http://example.com/codeflow?start=26&length=16)

I am not strictly sure this one is Medium.

Reply:
Ties into my other comment: we don't have an objective metric to define the confidence levels.